app/controllers/game.js

```diff
@@ -0,0 +1,20 @@
+var config = require('../../config/config');
+var mongoose = require('mongoose');
+var Game = mongoose.model('Game');
+
+exports.all = function(req, res) {
+	Game.find({}).exec(function(err, game) {
+		if (err) return next(err);
+		res.send(game);
```

Comment:
Think you should use res.json.
res.json eventually calls res.send, but before that it:

- respects the json spaces and json replacer app settings
- ensures the response will have utf8 charset and application/json content-type